src/formatter.ts

```diff
@@ -551,7 +503,8 @@ export class Formatter {
     return this.minTotalWidth;
   }
 
-  /** Calculate the resolution multiplier for `voices`. */
+  /** Calculate the resolution multiplier for `voices`, which is the
+   * least common multiple of all the voices' getResolutionMultiplier() results. */
   static getResolutionMultiplier(voices: Voice[]): number {
```

Comment:
Next PR I'll move this w/ the other static methods above, but it was already here.